python-package/lightgbm/callback.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, best_iteration, best_score):
 
 
 # Callback environment used by callbacks
-CallbackEnv = collections.namedtuple(
+CallbackEnv = collections.namedtuple( # type: ignore
```

Comment:
could you remove this? I think this fix in another PR will handle this: https://github.com/microsoft/LightGBM/pull/3916/files#r571475894

Reply:
```suggestion
CallbackEnv = collections.namedtuple(
```